lib/progress.c

```diff
@@ -229,16 +229,16 @@ void Curl_pgrsStartNow(struct Curl_easy *data)
  * need to wait until we're back under the speed limit, if needed.
  *
  * The way it works is by having a "starting point" (time & amount of data
- * transferred by then) used in the speed computation, to be used instead of the
+ * transfered by then) used in the speed computation, to be used instead of the
  * start of the transfer.
  * This starting point is regularly moved as transfer goes on, to keep getting
- * accurate values (instead of average over the entire transfer).
+ * accurate values (instead of average over the entire tranfer).
```

Comment:
yes that's again as wrong as before ( reverted my change to the file to original, as it broke travis - https://travis-ci.org/curl/curl/jobs/215231989 ) .. line-length issue

/usr/bin/perl ./checksrc.pl -D. -W./curl_config.h      \
	./*.[ch] ./vauth/*.[ch] ./vtls/*.[ch]
./progress.c:232:81: warning: Longer than 79 columns (LONGLINE)
  * transferred by then) used in the speed computation, to be used instead of the
./progress.c:237:81: warning: Longer than 79 columns (LONGLINE)
  * This function takes the current amount of data transferred, the amount at the

there are some more findings by my bot, but renaming variables, executable, .. I wont break it. 
it's not a 100% fix, as the dict is quite small. It run it with option 
" -D Enable rules based on lintian.debian.org ( git:ebac9a7, ~2300 )"